recipes/postgresql/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [ "$(uname)" == "Linux" ]; then
+  OPTS="--with-openssl"
+else
+    # At first attempt, openssl-enabled fails to build on Mac.  Needs further investigation.
+    OPTS=""
+fi
+
+./configure --prefix=$PREFIX \
+            --without-readline \
+            --with-libraries=$PREFIX/lib \
+            --with-includes=$PREFIX/include \
+            $OPTS
+
+make
+# make check # Failing with 'initdb: cannot be run as root'.
```

Comment:
@msarahan and @pelson any advice here? The test fail when we run as `root`. Maybe we should change the docker image to a "build" user by default. Not sure what would be the consequences of that though.